internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -428,6 +440,74 @@ func (r *ClusterDeploymentsReconciler) updateInstallConfigOverrides(ctx context.
 	return nil
 }
 
+func (r *ClusterDeploymentsReconciler) syncManifests(ctx context.Context, cluster *common.Cluster, manifests map[string]string, alreadyCreatedManifests models.ListManifests) error {
+	r.Log.Debugf("Going to sync list of given with already created manifests")
+
+	for _, manifest := range alreadyCreatedManifests {
```

Comment:
Actually, this implementation isn't correct.
You can't just compare the manifest file name.
You should validate the manifest content

Reply:
so didn't wanted to handle this case :(

Reply:
It will be very problematic one, not sure if it worth supporting it

Reply:
maybe will be enough to describe process of changing file content? cause it will require to run file download for each file. I think we shouldn't support it (too expensive)